### PR TITLE
fix: use sites feature in django admin for setting site url and name 

### LINKED ISSUE #434: configure site url and site name in one place
Right now, site name and site url, e.g. `Jyväskylä` and `mulysa.jkl.hacklab.fi` are configured in two places: in `settings.py` (or possibly in the constance admin panel config page, see #433) and in the admin panel through the [Sites framework](https://docs.djangoproject.com/en/4.1/ref/contrib/sites/). If we want to avoid having to configure this in two places we need to choose one mechanism for configuring this value.

These two differ slightly. settings.py/Constance is set up so that there is one site name and site url set for a given deployment of mulysa. The Sites framework is designed to facilitate multiple domains, such as in this example cited by Django:

> The [LJWorld.com](https://www2.ljworld.com/) and [Lawrence.com](http://www.lawrence.com/) sites are operated by the same news organization – the Lawrence Journal-World newspaper in Lawrence, Kansas. LJWorld.com focused on news, while Lawrence.com focused on local entertainment. But sometimes editors wanted to publish an article on both sites.
>
> The naive way of solving the problem would be to require site producers to publish the same story twice: once for LJWorld.com and again for Lawrence.com. But that’s inefficient for site producers, and it’s redundant to store multiple copies of the same story in the database.
>
> A better solution removes the content duplication: Both sites use the same article database, and an article is associated with one or more sites.

It's not immediately clear how this kind of thing would be useful for mulysa. However, in its most basic form, the sites framework could enable hacklabs to use multiple domains without any difference between the services available at each domain. Their mulysa deployment could be accessible at both `mulysa.jkl.hacklab.fi` and `mulysa.ninja` or whatever.

However, fully adopting this feature would require a bit of refactoring. Currently, in the code, the site url and name from `settings.py` is nearly ubiquitous. 

## Difference in usage

Accessing the site url or sitename is pretty much the same level of convenience:

With `settings.py` (or `settings_local.py`)
```py
from drfx import settings

domainFromSettings = settings.SITE_URL
```
With Constance:
```py
from constance import config

domainFromConstance = config.SITE_URL
```
With Sites:
```py
from django.contrib.sites.models import Site

domainFromSites = Site.objects.get_current().domain 
```

With constance (see #433), it would probably be more elegant if we placed these settings in the constance settings together with other similar things like address, links to association rules, etc. It's a bit confusing to have to configure the URL and hacklab name in a separate place.

I suggest we remove the sites feature since it's very unlikely that any hacklab will want to set up multiple urls. Putting all the site-specific settings in one place (constance) is more intuitive more convenient both in terms of UX and DX.

### PR DESCRIPTION
**Pull request by braaar**
fixes #434

### CODE REVIEW

**Comment by braaar** [comment]
You can see the difference between this and #433 by visiting [this link](https://github.com/HacklabJKL/mulysa/compare/constance-config...HacklabJKL:mulysa:sites)

**Comment by braaar** [comment]
@olmari Have you had time to go through this yet? Want me to record a video demo of this feature?

**Comment by olmari** [comment]
Makes it easier (to remember) if review is requested, self-requested now

**Comment by braaar** [comment]
Corrected some irrelevant formatting changes in banktransaction.html that broke the template code blocks

File: www/templates/www/banktransaction.html [hunk 1]
```diff
@@ -1,54 +1,48 @@
...
-    {{ settings.RECEIPTSTREET}}<br />
-    {% trans 'Business ID' %}: {{ settings.RECEIPTREGID}}<br />
+    {{config.RECEIPT_NAME }}<br />
+    {{config.RECEIPT_ADDRESS}}<br />
+    {% trans 'Business ID' %}: {{config.RECEIPT_REGID}}<br />
   </p>
   <p>
-    {% get_verbose_field_name banktransaction "date" %}: {{ banktransaction.date }}<br />
+    {% get_verbose_field_name banktransaction "date" %}: {{ banktransaction.date
+    }}<br />
```
[the remark this answers is not in the document]
Comment (olmari) [reply www/templates/www/banktransaction.html:14]: This was fixed in 433 but we are now waiting for merging it to this PR

[the remark this answers is not in the document]
Comment (braaar) [reply www/templates/www/banktransaction.html]: This is fixed now

**Comment by tswfi** [comment]
@braaar can you rebase this?

File: www/templates/www/banktransaction.html [hunk 2]
```diff
@@ -1,8 +1,5 @@
-{% extends "www/base.html" %}
-{% load i18n %}
-{% load bootstrap4 %}
-{% block content %}
-{% load verbose_names %}
+{% extends "www/base.html" %} {% load i18n %} {% load bootstrap4 %} {% block
+content %} {% load verbose_names %}
```
Comment (tswfi) [inline www/templates/www/banktransaction.html:2]: Can we keep these on separate lines? (I'm guessing your IDE did some autoformatting here)

**Comment by braaar** [comment]
Reverted the formatting on the HTML file now

**Pull request merged**